src/main/java/de/tum/in/ase/parser/strategy/SpotbugsParser.java

```diff
@@ -28,6 +30,11 @@ public Report parse(Document doc) {
         // Element BugCollection
         Element root = doc.getRootElement();
 
+        String sourceDirectory = root.getFirstChildElement(PROJECT_ELEMENT).getFirstChildElement(SOURCE_DIRECTORY_ELEMENT).getValue();
```

Comment:
Please make this null safe. `getFirstChildElement` might return null. This should never happen but Spotbugs might change their reporting format in the future.